tests/test_urs_util.py

```diff
@@ -110,13 +110,24 @@ def test_get_urs_url(mock_get_urs_creds, context):
 @mock.patch(f"{MODULE}.urllib.request", autospec=True)
 def test_get_profile(mock_request):
     mock_response = mock.NonCallableMock()
-    mock_response.read.return_value = '{"foo": "bar"}'
+    mock_response.read.return_value = ('{'
+                                       '"uid": "user_id",'
+                                       '"first_name": "John",'
+                                       '"last_name": "Smith",'
+                                       '"email_address": "peter.l.smith@nasa.gov",'
+                                       '"user_groups": []'
+                                       '}'
+                                       )
```

Comment:
Might be better just to use `json.dumps` here with a dictionary.
```suggestion
    mock_response.read.return_value = (
        '{'
        '"uid": "user_id",'
        '"first_name": "John",'
        '"last_name": "Smith",'
        '"email_address": "peter.l.smith@nasa.gov",'
        '"user_groups": []'
        '}'
    )
```